vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewFragment.kt

```diff
@@ -96,6 +112,12 @@ class SessionOverviewFragment :
                 ?.setTitle(titleResId)
     }
 
+    private fun updateVerifyButton(deviceId: String) {
+        views.sessionOverviewInfo.viewVerifyButton.debouncedClicks {
+            viewModel.handle(SessionOverviewAction.VerifySession(deviceId))
```

Comment:
I understand that this Fragment is displaying only one session. So passing the device Id here is probably useless, the ViewModel already knows it.
I would do some cleanup.

Reply:
Yes removed here: [4f3df2c](https://github.com/vector-im/element-android/pull/7139/commits/4f3df2c213ba4325ab43201e702eb36200fd3009). Thanks for the review!